cstar/job.py

```diff
@@ -86,11 +88,18 @@ def get_cluster_topology(self, seed_nodes):
         count = 0
         tried_hosts = []
         for host in seed_nodes:
+            print("JMX USERNAME =", self.jmx_username)
             tried_hosts.append(host)
             conn = self._connection(host)
 
-            describe_res = conn.run(("nodetool", "describecluster"))
-            topology_res = conn.run(("nodetool", "ring"))
+            if self.jmx_username and self.jmx_password:
+                print("USING JMX AUTH")
+                describe_res = conn.run(("nodetool", "-u", "cassandra", "-pw", "cassandra", "describecluster"))
+                topology_res = conn.run(("nodetool", "-u", "cassandra", "-pw", "cassandra", "ring"))
```

Comment:
These aren't the only nodetool calls made by cstar.
You have others [here](https://github.com/spotify/cstar/blob/65cbb49f5203ff231fbda6702472794a14c323a2/cstar/job.py#L141) and [here](https://github.com/spotify/cstar/blob/65cbb49f5203ff231fbda6702472794a14c323a2/cstar/job.py#L161).

Reply:
Also, I'm confused. Shouldn't this say `"-u", self.jmx_username, "-pw", self.jmx_password`?

Reply:
>Shouldn't this say `"-u", self.jmx_username, "-pw", self.jmx_password`?

It should.